checkers/ruleguard_checker.go

```diff
@@ -135,20 +135,41 @@ func newRuleguardChecker(info *linter.CheckerInfo, ctx *linter.CheckerContext) (
 
 	enabledGroups := make(map[string]bool)
 	disabledGroups := make(map[string]bool)
+	enabledTags := make(map[string]bool)
+	disabledTags := make(map[string]bool)
 
 	for _, g := range strings.Split(info.Params.String("disable"), ",") {
 		g = strings.TrimSpace(g)
+		if t := strings.Split(g, "#"); len(t) == 2 {
+			disabledTags[t[1]] = true
+			continue
+		}
+
 		disabledGroups[g] = true
 	}
 	flagEnable := info.Params.String("enable")
 	if flagEnable != "<all>" {
 		for _, g := range strings.Split(flagEnable, ",") {
 			g = strings.TrimSpace(g)
+			if t := strings.Split(g, "#"); len(t) == 2 {
+				enabledTags[t[1]] = true
```

Comment:
How about?
```
if strings.HasPrefix(g, "#") {
  enabledTags[strings.TrimPrefix(g, "#")] = true
}
```
It looks like you're splitting a string like `#tag` just to check if it will result in 2 elements, making `foo#tag` a valid input.

Reply:
fixed